### PR TITLE
docs: include all tests in README.md

### PR DESCRIPTION
**Pull request by addievo**
### Description
Adding usage examples for all types of streams to readme

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -13,3 +13,4 @@
 /build
 /builds
 /dist/tsbuildinfo
+/images
```
Comment (CMCDragonkai) [inline .npmignore:16]: Scratch this. I don't think it should be done.

File: README.md [hunk 2]
```diff
@@ -675,82 +587,315 @@ the client sends a single request and receives multiple responses in a read-only
...
+const squaredStream = await rpcClient.methods.SquaredNumbers(235);
+const outputs: Array<number> = [];
+for await(const num of squaredStream) {
+outputs.push(num);
+}
+console.log(`Squared numbers are: ${outputs.join(', ')}`);
+} catch (error) {
+console.error("Error in execute:", error);
+}
+}
```
Comment (CMCDragonkai) [inline README.md:717]: Hey can you make sure code is properly indented with 2 space indent?

Make sure the coding style conforms with our style.

File: images/duplexTest.png
Comment (CMCDragonkai) [inline images/duplexTest.png:1]: I don't think you need to do this. I just wanted you to test that it worked, you do not need these images.

File: images/unaryTest.png
Comment (CMCDragonkai) [inline images/unaryTest.png:1]: Same here, don't need this image at all.

File: .npmignore (hunk 1 above)
Comment (addievo) [reply .npmignore:16]: 👍🏻

File: README.md (hunk 2 above)
Comment (CMCDragonkai) [reply README.md:717]: In fact even better if you were to put the code into somewhere, and apply the style linting to it to fix it.

Comment (CMCDragonkai) [reply README.md:717]: Then copy onto the `README.md`.

**Pull request merged**